File: 04_Thu/day4.py
# Ask the user for two numbers (x, y)

# Print the times tables up to x * y

# Each line should look like the following example: 

#  2 * 3 = 6

def get_num():
    # num1 = int(input("Give me a number: "))
    # num2 = int(input("Give me another number: "))

    # for num in range(num2 + 1):
    #     total = num * num2
    #     if num > 0:
    #         print(f"{num} * {num2} = {total}")

    num1 = int(input("Give me a number: "))
    num2 = int(input("Give me another number: "))

    n = 1
    while (n < num2):
        total = n * num2
        print(f"{n} * {num2} = {total}")

        n += 1


def main():
    get_num()


if __name__ == "__main__":
    main()

File: 04_Thu/day4.js
// function ex() {
//     var num1 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
//     var num2 = [1, 2, 3, 4]


//     num1.forEach(function (element) {
//         if (element != 0) {
//             var num = num2.length;
//             total = element * num
//             console.log(`${element} * ${num} = ${total}`)
//         }
//     })
// }

function ex() {
    num1 = 24
    num2 = 1;

    while (num2 < num1) {
        var total = num2 * num1
        console.log(`${num2} * ${num1} = ${total}`)

        num2 += 1
    }
}

ex()

